src/MessagePack/SequenceReaderExtensions.cs

```diff
@@ -54,24 +54,6 @@ internal static partial class SequenceReaderExtensions
             return true;
         }
 
-        /// <summary>
-        /// Reads an <see cref="sbyte"/> from the next position in the sequence.
-        /// </summary>
-        /// <param name="reader">The reader to read from.</param>
-        /// <param name="value">Receives the value read.</param>
-        /// <returns><c>true</c> if there was another byte in the sequence; <c>false</c> otherwise.</returns>
-        public static bool TryRead(ref this SequenceReader<byte> reader, out sbyte value)
```

Comment:
Made this file identical to https://github.com/dotnet/corefx/blob/master/src/System.Memory/src/System/Buffers/SequenceReaderExtensions.Binary.cs and moved these additional extensions to a separate file. Makes it easier to compile for netcoreapp3.0

Reply:
Given we're keeping our copy of `SequenceReader<T>`, I think we're going to need to keep the extension methods on our type as well, so is there still value in splitting these into two separate files?